### DISCUSSION TITLE
翻译的首选要素

### DISCUSSION THREAD

**Discussion by FarmerChillax** [apiflask/docs-zh#28] [General]
不可否认翻译最好能做到 **信达雅**，但更多的时候需要在翻译的准确性与可读性之间做出抉择。因此希望讨论一下翻译的时候，应该 **可读性** 优先还是 **准确性** 先行，亦或是别的方案。

**Comment by FarmerChillax** [comment]
个人认为在 Quick start 之类的章节应该适当的照顾萌新的入门门槛，在翻译上尽量采用通俗易懂的语句来讲解，并且将一些专业的词汇也做一定的解释？

但这对翻译来讲，会可能与原文直译产生较大的分歧。

对于进阶、 API 介绍等章节则尽可能的使用严谨的词汇来描述？

比如在 **基础用法** 章节中，有这么一段：
```markdown
If the validation passed, the data will inject into the view function as
a positional argument in the form of `dict`. Otherwise, an error response
with the detail of the validation result will be returned.
```

翻译一：
```markdown
如果验证通过，请求体的数据将会以 `dict` 的形式**输出**到视图函数参数的固定位置中。**否则就把验证结果的详细信息以错误响应的形式返回出去。**
```

翻译二：
```markdown
如果验证通过，请求体的数据将会以 `dict` 的形式**注入**到视图函数参数的固定位置中。否则，**错误响应将返回验证结果的详细信息。**
````

使用 **注入** 这一词汇，则会让没有一定基础的人产生迷惑。

**Reply by greyli** [reply]
可读性优先，可以适当做一些改写和补充。英文原文其实有些地方我可能没有表达清楚或是没有使用最佳词汇（受限于英文水平……）。
